### PR TITLE
CHK-806: Tests: Address with no zipcode

### PR DESCRIPTION
**Pull request by silviasfon**
#### What is the purpose of this pull request?
Testing that the zipcode field appears on

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: tests/shipping/models/Delivery - No zipcode.model.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        'Corrientes 240, Las Varillas, Córdoba, Argentina'
+      )
+
+      cy.get('.pac-item')
+        .first()
+        .trigger('mouseover')
+
+      cy.get('.pac-item')
+        .first()
+        .click()
```
Comment (lucasecdb) [inline tests/shipping/models/Delivery - No zipcode.model.js:41]: what do you think of intercepting the gmaps autocomplete response here to make sure it won't return a postal code?

**Review by lucasecdb — APPROVED** [review]
nice! ✨

File: tests/shipping/models/Delivery - No zipcode.model.js [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+} from '../../../utils/shipping-actions'
+import {
+  completePurchase,
+  fillBillingAddress,
+  payWithCreditCard,
+} from '../../../utils/payment-actions'
+import { SKUS } from '../../../utils/constants'
+
+export default function test(account) {
+  describe(`Delivery - Credit Card - ${account}`, () => {
```
Comment (wdsrocha) [inline tests/shipping/models/Delivery - No zipcode.model.js:20]:
```suggestion
  describe(`Delivery - No zipcode - ${account}`, () => {
```

File: utils/payment-actions.js [hunk 3]
```diff
@@ -72,13 +72,38 @@ export function fillCreditCardInfo(
...
+
+    if (options.street !== undefined)
+      cy.wrap($body)
+        .find(`#payment-billing-address-street-${options.id || '0'}`)
+        .type(options.street, { force: true })
+
+    if (options.city !== undefined)
+      cy.wrap($body)
+        .find(`#payment-billing-address-city-${options.id || '0'}`)
+        .select(options.city)
```
Comment (wdsrocha) [inline utils/payment-actions.js:106]: Is there any reason on why the last select doesn't need `force` option?
```suggestion
        .select(options.city,  { force: true })
```

File: utils/payment-actions.js [hunk 4]
```diff
@@ -72,13 +72,38 @@ export function fillCreditCardInfo(
...
+    id: 0,
+  }
+) {
+  queryIframe($iframe => {
+    const $body = getIframeBody($iframe)
+
+    // We type with force:true because of https://github.com/cypress-io/cypress/issues/5830
+    if (options.postalCode !== undefined)
+      cy.wrap($body)
+        .find(`#payment-billing-address-postalCode-${options.id || '0'}`)
```
[the remark this answers is not in the document]
Comment (wdsrocha) [reply utils/payment-actions.js:90]: That's just a nitpick though 👀

File: utils/payment-actions.js (hunk 3 above)
Comment (silviasfon) [reply utils/payment-actions.js:106]: I copied this from the fillCreditCardInfo function and apparently this kind of field doesn't need the force option, only text fields

Comment (wdsrocha) [reply utils/payment-actions.js:106]: ohh, its a select! Didn't noticed it. LGTM then 😄

File: utils/payment-actions.js (hunk 4 above)
[the remark this answers is not in the document]
Comment (silviasfon) [reply utils/payment-actions.js:90]: can't you send an object with other fields but undefined id field?

[the remark this answers is not in the document]
Comment (wdsrocha) [reply utils/payment-actions.js:90]: that's true!

but in order to reduce some verbosity, we could just create an `id` variable outside (again, not required)
```suggestion
    const id = options.id || '0'
    // some code in between...
        .find(`#payment-billing-address-postalCode-${id}`)
```

**Review by wdsrocha — APPROVED** [review]
gj! ;)

**Pull request merged**